### PR TITLE
null safety

### PR DESCRIPTION
**Pull request by hovadur**

### CODE REVIEW

File: example/lib/main.dart [hunk 1]
```diff
@@ -21,18 +22,24 @@ class MyApp extends StatelessWidget {
...
   _MyHomePageState createState() => _MyHomePageState();
 }
 
 class _MyHomePageState extends State<MyHomePage> {
   GlobalKey<FormState> _oFormKey = GlobalKey<FormState>();
-  TextEditingController _controller1;
-  TextEditingController _controller2;
-  TextEditingController _controller3;
-  TextEditingController _controller4;
+  TextEditingController _controller1 =
```
Comment (harsimranmaan) [inline example/lib/main.dart:33]: This could be qualified with `late` keyword to avoid early initialization

Comment (hovadur) [reply example/lib/main.dart]: done

**Comment by grahamsmith** [comment]
@hovadur thanks for working on this 👍  Is there any remaining work to get this merged and released? If so happy to help :)

**Comment by hslbetto** [comment]
Thanks for all your contribution

**Pull request merged**